modules/xfem/tests/diffusion_xfem/tests

```diff
@@ -7,4 +7,13 @@
     # XFEM requires --enable-unique-ids in libmesh
     unique_id = true
   [../]
+  [./diffusion_xfem_flux_bc]
+    type = Exodiff
+    input = diffusion_flux_bc.i
+    exodiff = 'diffusion_flux_bc_out.e'
+    map = false
+    # XFEM requires --enable-unique-ids in libmesh
```

Comment:
This comment is probably not necessary, especially since it's technically not correct `--enable-unique-id` is the exact name of the flag.